entgql/annotation.go

```diff
@@ -62,6 +64,13 @@ type (
 
 	// SkipMode is a bit flag for the Skip annotation.
 	SkipMode int
+
+	QueryFieldConfig struct {
```

Comment:
```suggestion
	FieldConfig struct {
```
Allow using it in other types in the future (e.g. Mutation).